docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,27 @@
+# Tutorial: Deploy the Gateway API integrator charm
+
+## What you'll do
+This tutorial will walk you through deploying the gateway-api-integrator charm, you will:
```

Comment:
Nit-pick: Change the comma to a semicolon and capitalise "deploy"

Reply:
updated, thanks!